custom_components/google_home/translations/en.json

```
@@ -11,7 +11,7 @@
       }
     },
     "error": {
-      "auth": "Username/Password is incorrect. If your google account has 2FA enabled please generate app password (https://myaccount.google.com/apppasswords)"
+      "auth": "Username/Password is incorrect. If your google account has 2FA enabled please generate app password (https://myaccount.google.com/apppasswords). If authentication still failes please see https://github.com/leikoilja/ha-google-home/issues/95"
```

I think we should use a link to a troubleshooting section. Do you want to complete #113?
```suggestion
      "auth": "Username/Password is incorrect. If your google account has 2FA enabled please generate app password (https://myaccount.google.com/apppasswords). If authentication still fails please see https://github.com/leikoilja/ha-google-home/issues/95"
```

---

Oh thanks 💥

---

Sorry, @KapJI, haven't seen your comment. Will check #113 and change the link here to point to troubleshooting section